### PR TITLE
Updates more arch diagrams and updates federated ETL description

### PR DESCRIPTION
**Pull request by bstuder99**
I updated several of our architecture diagrams to look more visually similar and easier to understand. I also updated an outdated diagram for Federated ETL and updated to corresponding doc to match the description

### CODE REVIEW

File: images/diagrams/fed-etl-agg-arch.png
Comment (thomasvn) [inline images/diagrams/fed-etl-agg-arch.png:1]: Could we simplify "Queried federated data"? Maybe replace with "Multicluster Kubecost data"? Really, it just needs to encapsulate the fact that both your UI and your API calls will now return data about all your monitored clusters.

**Comment by thomasvn** [comment]
@bstuder99 Looking good! Thanks for your work standardizing these diagrams, and also updating their corresponding text.

File: images/diagrams/fed-etl-agg-arch.png
Comment (bstuder99) [reply images/diagrams/fed-etl-agg-arch.png:1]: Yeah thats a better way of wording it. Was mainly trying to adhere to wording from the original diagram but I like this suggestion

**Comment by chipzoller** [comment]
@thomasvn, are we good to merge this into `main` at this time?

**Review by thomasvn — APPROVED** [review]
Yes, let's go ahead and merge!

**Pull request merged**